backend/src/main/java/com/senior/project/backend/domain/Student.java

```diff
@@ -0,0 +1,20 @@
+package com.senior.project.backend.domain;
+
+import lombok.*;
+
+import java.util.Date;
+
+@Data
+@ToString
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Student {
+	
+    private String id;
+	
+	private int userID;
```

Comment:
It looks like a consistent thing, so scanning through the domain classes to fix it would be worth it. Low priority though, its just styling